client/state/signup/progress/schema.ts

```diff
@@ -26,7 +26,7 @@ export interface StepState {
 		url: string;
 	};
 	lastUpdated: number;
-	providedDependencies?: string[];
```

Comment:
So this type was wrong from the very beginning? As this is actually something like `{ [key: string]: any }`? 🥲

Reply:
Oh wow. I'm surprised we weren't seeing runtime errors because the data in `localeStorage` was the wrong type. But I see that the json schema declaration above never actually included a definition for the `providedDependencies` property. So we were getting away with it 😮

Reply:
Well, TypeScript is never checked at runtime 😄 This kind of "bug" has bitten me a few times in my past projects as well.

Reply:
Oh, I see what you mean now. Right right, we were so lucky. 😂